generators/server/templates/src/main/java/package/domain/_Authority.java

```diff
@@ -29,6 +29,13 @@
     @Column(length = 50)<% } %>
     private String name;
 
+    public Authority() {
```

Comment:
For me the error is with the other constructor, as we need an empty one anyway, and it's the other one that lacks the validation.... I'm correcting the Sonar error, but indeed it might not be needed at all - this was added by @cbornet so he probably knows better